src/utils/variables.scss

```diff
@@ -0,0 +1,16 @@
+$-color-background: rgb(112, 166, 224);
+$-color-shadow: rgba(232, 232, 240, 0.64);
+$-color-background-todo: rgb(228, 245, 246);
+$-color-white: white;
+$-color-error: red;
+$-color-text-gray: gray;
+$-color-completed: #080;
+$-color-unCompleted: #f00;
+
+$-border-radius: 8px;
+
+$-transition: 0.5s;
+
+$-margin: 20px;
+
+$-field-heigth: 42px;
```

Comment:
Consider using a more descriptive variable name. `$-field-height` is not very descriptive. Consider using something like `$input-height` or `$button-height`. 🎨